--- netpanzer-0.1.5/netpanzerdata-0.1.3/configure.ac.orig	2003-11-29 17:27:54.000000000 +0100
+++ netpanzer-0.1.5/netpanzerdata-0.1.3/configure.ac	2004-07-19 21:17:03.791359336 +0200
@@ -118,8 +118,9 @@
 	[AC_MSG_ERROR([Please install SDLImage >= 1.2.1])],
 	[$SDL_CFLAGS], [$SDL_LIBS])
 
-NP_FINDLIB([PHYSFS], [physfs], [physfs >= 1.9],
+NP_FINDLIB([PHYSFS], [physfs], [physfs >= 0.1.9],
 	NP_LANG_PROGRAM([#include <physfs.h>
+#if PHYSFS_VER_MAJOR < 1
 #if PHYSFS_VER_MINOR < 1
 # error PHYSFS is too old
 #else
@@ -128,6 +129,7 @@
 #   error PHYSFS is too old
 #  endif
 # endif
+#endif
 #endif]),
 	[], [-lphysfs],
 	[],
